docs/developer-guide/syntax-reference.rst

```diff
@@ -1581,6 +1583,8 @@ Scalar functions
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
 | LEN                    |  ``LEN(col1)``                                                            | The length of a string.                           |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| LOG                    |  ``LOG(col1)``                                                            | The natural logarithm of a value.                         |
```

Comment:
I think you have a too many spaces for `|`

Reply:
done.